.gitignore

```diff
@@ -31,6 +31,3 @@ node_modules
 *.swo
 *.swp
 *.swn
-
-# compiled
-build
```

Comment:
Are you sure you meant to do this?

Reply:
Ahh right, I didn't... My mistake. I wanted to `npm install` it directly from my fork but I pushed it all to the same branch. I'll close this PR and create a new one with just that one commit.